hardware/Layout-Desing-Guide.md

```diff
@@ -0,0 +1,486 @@
+
+<details>
+
+<summary><font size=5>Table of Contents</font> </summary>
+ 
+ 
+ - [1. Why is the proper layout design important?](#1.-why-is-the-proper-layout-design-important?)
+ - [2. RF section](#2.-rF-section)
+    - [2.1. Different parts of RF section](#2.1.-Different-parts-of-RF-section)
+      - [2.1.1. Function of RF matching  and filtering network](#2.1.1.-Function-of-RF-matching-and-filtering-network)
+      - [2.1.2. Function of VDD filterings](#2.1.2.-Function-of-VDD-filterings)
+    - [2.2. Layout of RF section](#2.2.-Layout-of-RF-section)  
+      - [2.2.1. Layout of the matching network](#2.2.1.-Layout-of-the-matching-network)
+  - [3. Layout Design Practices](#3.-Layout-Design-Practices)
+    - [3.1. Layout Design Practices - 2.4 GHz matching network](#3.1.-Layout-Design-Practices---2.4-GHz-matching-network)
+    - [3.2. Layout Design Practices - Sub-GHz matching network](#3.2.-Layout-Design-Practices---Sub-GHz-matching-network)
+    - [3.3. Layout Design Practices - HFXO](#3.3.-Layout-Design-Practices---HFXO)
+    - [3.4. Layout Design Practices - VDD filtering](#3.4.-Layout-Design-Practices---VDD-filtering)
+    - [3.5  Layout Design Practices - General](#3.5.-Layout-Design-Practices---General)
+  - [4. Quiz](#4.-Quiz)
+
+</details>
+
+
+
+
+The purpose of this material is to help users design PCBs
+for the EFR32 Wireless Gecko Portfolio using design practices
+that allow for good RF performance.
+
+PCB layout requirements are described in the application note, [AN928.1: EFR32 Series 1 Layout Design Guide](https://www.silabs.com/documents/public/application-notes/an928.1-efr32-series1-layout-design-guide.pdf),  and the layout design process for EFR32 Series 1 Modules is discussed in [the datasheet of each module](https://www.silabs.com/support/resources.ct-data-sheets.ct-miscellaneous.p-wireless_bluetooth-low-energy_blue-gecko-bluetooth-low-energy-modules.p-wireless_bluetooth-low-energy_blue-gecko-bluetooth-low-energy-modules_bgm11s12f256ga-v2.p-wireless_bluetooth-low-energy_efr32bg1-series-1-modules.p-wireless_bluetooth-low-energy_efr32bg13-series-1-modules.sort=2,asc) and  besides that EFR32 reference design files are available in Simplicity Studio.
+
+
+# 1. Why is the proper layout design important? 
+
+If the layout is an inappropriate design that can cause 
++ Degradation in the communication distance 
++ The unit can violate the relevant EMC regulation (high harmonics, spur) 
++ The battery life can be reduced
+
+# 2. RF section 
+
+
+## 2.1. Different parts of RF section
+The most important part of the proper layout design is the RF circuitry itself.
+
+In the case of dual band RFIC, this RF part includes the following:
+1. Sub-GHz and 2.4 GHz RF matching newtork  
+2. Power supply filterings 
+3. High frequency crystal  
+
+
+
+<p align="center">
+  "<img src="files/Layout-Design-Guide/matching network.png">
```

Comment:
This image looks to be from an MG12 radio board with BGA package, which is larger and more complicated than the QFN48 package.  It might be worth replacing this MG12 BGA schematic image with an MG13 QFN48 schematic image to simplify it a bit, as it would no longer have all of the ground pins from the BGA package cluttering the pin-out.  QFN48 package is more commonly used than BGA package as well, which is another reason to make this change.

Reply:
It's changed to QFN48.